_posts/en/newsletters/2019-06-12-newsletter.md

```diff
@@ -0,0 +1,76 @@
+---
+title: 'Bitcoin Optech Newsletter #50'
+permalink: /en/newsletters/2019/06/12/
+name: 2019-06-12-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+*None this week.*
+
+## News
+
+- FIXME: core dev transcripts
+  http://diyhpl.us/wiki/transcripts/bitcoin-core-dev-tech/2019-06-*
+
+  See also: https://github.com/bitcoin-core/bitcoin-devwiki/wiki/P2P-Design-Philosophy
+
+- FIXME: Taproot executive briefing video
+
+- FIXME (maybe) RBF proposal, bitcoin-dev ml
+
+- FIXME (maybe) blind statechains, bitcoin-dev ml
+
+- FIXME issue 50 note
+
+## Bech32 sending support
+
+*Week 13 of 24 in a [series][bech32 easy] about allowing the people
+you pay to access all of segwit's benefits.*
+
+{% comment %}<!-- weekly reminder for harding: check Bech32 Adoption
+wiki page for changes -->{% endcomment %}
+
+{% include specials/bech32/13-adoption-speed.md %}
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[LND][lnd repo], [C-Lightning][c-lightning repo], [Eclair][eclair repo],
+[libsecp256k1][libsecp256k1 repo], and [Bitcoin Improvement Proposals
+(BIPs)][bips repo].*
+
+- [LND #2802][] allows LND to calculate the probability that a
```

Comment:
Joost talked about this in his Breaking Bitcoin presentation. Perhaps we could link to the transcript here: http://diyhpl.us/wiki/transcripts/breaking-bitcoin/2019/lightning-network-routing-security/